### PR TITLE
docs(samples): fixed drag & drop errow in kanban sample

### LINKED ISSUE #3514: Errors thrown when dragging elements in same group for Drag&Drop sample
### Description  
Errors are thrown when dragging a card inside the same group.

### Steps to reproduce  

1. Open https://www.infragistics.com/angular-demos/interactions/kanban-sample
2. In the `To Do` section drag a card up/down to go over another card
3. Observe the console

### Result  
Errors are thrown:

TypeError: Cannot read properties of undefined (reading 'nativeElement')
    at s.onItemLeave (711.8afdec845d50a74f.js:1:37767)

### Expected result  
There should be no errros

### PR DESCRIPTION
**Pull request by tishko0**
closes #3514

### CODE REVIEW

File: src/app/interactions/drag-drop/kanban-sample/kanban-sample.component.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-shadow */
 /* eslint-disable @typescript-eslint/naming-convention */
-import { ChangeDetectorRef, Component, ElementRef, OnInit, Renderer2, ViewChild } from '@angular/core';
+import { ChangeDetectorRef, Component, ElementRef, OnInit, Renderer2, viewChild, ViewChild } from '@angular/core';
```
Comment (dkamburov) [inline src/app/interactions/drag-drop/kanban-sample/kanban-sample.component.ts:3]:
```suggestion
import { ChangeDetectorRef, Component, ElementRef, OnInit, Renderer2, ViewChild } from '@angular/core';
```

**Review by ChronosSF — CHANGES_REQUESTED** [review]
Fixes going to prod are never only merged to master. You either make a PR to vnext and master, or just vnext if a mass merge is possible.

File: src/app/interactions/drag-drop/kanban-sample/kanban-sample.component.ts (hunk 1 above)
Comment (ChronosSF) [reply src/app/interactions/drag-drop/kanban-sample/kanban-sample.component.ts:3]: Agree with Deyan. Why is this imported with lower case too?

**Review by ChronosSF — CHANGES_REQUESTED** [review]

File: src/app/interactions/drag-drop/kanban-sample/kanban-sample.component.ts (hunk 1 above)
Comment (tishko0) [reply src/app/interactions/drag-drop/kanban-sample/kanban-sample.component.ts]: probably imported by mistake due to good intellisense and I missed it. Removed it now.

**Review by dkamburov — APPROVED** [review]

**Review by ChronosSF — APPROVED** [review]

**Pull request merged**